src/lib/configuration.rs

```diff
@@ -0,0 +1,49 @@
+use std::env;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    pub static ref NUN_USER: String = expect_env_var("NUN_USER", "mateus", false);
+    pub static ref NUN_PWD: String = expect_env_var("NUN_PWD", "mateus", false);
+    pub static ref NUN_DBS_DIR: String = expect_env_var("NUN_DBS_DIR", "/tmp/data/nun_db", false);
```

Comment:
Checkout the Pipeline https://github.com/mateusfreira/nun-db/runs/5138892658?check_suite_focus=true

Some tests are falling because of this default value it use to be `dbs`. Can you change it back to Dbs?